handlers/handler-bizkeeper/src/main/java/io/servicecomb/bizkeeper/Configuration.java

```diff
@@ -114,7 +114,7 @@ public boolean getIsolationTimeoutEnabled(String type, String microserviceName,
   public int getIsolationMaxConcurrentRequests(String type, String microserviceName,
       String qualifiedOperationName) {
     int concurrentRequests;
-    String p = getProperty("10",
+    String p = getProperty("1000",
```

Comment:
all configuration items convert to string, and then convert to int/boolean and so on
and this convert happened in every request, it's not good
can we create a new issue to optimize this?

Reply:
Please create JIRA for it.